src/pages/docs/infrastructure/workers/dynamic-worker-pools.md

```diff
@@ -71,12 +71,11 @@ You should validate that your deployments and runbooks work with the new version
 
 ### Modifying the worker pool 
 
-If the Worker Pool has been configured to specifically use a deprecated worker type, you will need to update the Worker Image on the Worker Pool.
+If the worker pool has been configured to specifically use a deprecated worker type, you will need to update the worker image on the worker pool.
 
-The Worker Pool with a deprecated Worker Type will show a `Deprecated` label next to the worker pool, available by navigating to **Infrastructure ➜ Worker Pools**:
-![Worker Pool list with deprecated worker](/docs/infrastructure/workers/images/deprecated-worker-pool-overview.png)
```

Comment:
I've removed this image altogether as there is no `Deprecated` image to take an example screenshot of. It's also of low utility, since there would be a vanishingly small number of customers who would not already have migrated off Ubuntu1804 or Windows2019 worker images.